BTrees/check.py

```diff
@@ -442,17 +442,17 @@ def visit_bucket(self, obj, path, parent, is_mapping,
 def check(btree):
     """Check internal value-based invariants in a BTree or TreeSet.
 
-    The btree._check() method checks internal C-level pointer consistency.
-    The check() function here checks value-based invariants:  whether the
+    The :meth:`~BTrees._base.btree._Tree._check` method checks internal C-level pointer consistency.
```

Comment:
`_base` and `_Tree` are implementation details that aren't published anywhere (and which only apply to the Python implementation). It would be fine to just express this as code:
```
The ``btree._check()``…
```